x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/queries/SemanticQueryBuilder.java

```diff
@@ -231,7 +232,7 @@ private SemanticQueryBuilder doRewriteGetInferenceResults(QueryRewriteContext qu
                 List.of(query),
                 Map.of(),
                 InputType.SEARCH,
-                InferModelAction.Request.DEFAULT_TIMEOUT_FOR_API,
```

Comment:
Should we create an additional constructor in `InferenceAction.Request` that does not include timeout, and use there the default one? That way we would be aligned with the default timeout and understand a potential change to the default.

Otherwise, I think it would be a good idea to target `InfreenceAction.Request.DEFAULT_TIMEOUT` here so we get automatic updates here if that changes.